api.swagger.yaml

```diff
@@ -1845,6 +1845,32 @@ paths:
           description: The UUID or request body JSON was malformed.
         '404':
           description: No manual page with that UUID was found.
+  '/manuals/{manual_page_id}/pdf-export':
```

Comment:
This (along with other comments below) implies that the node must be a manual page - this is not necessarily the case, we should also handle policies (and possibly other content types in the future)

Reply:
Thanks; fixed.